Cargo.toml

```diff
@@ -34,6 +34,8 @@ bytes = { workspace = true }
 clap = { workspace = true, features = ["derive"] }
 displaydoc = { workspace = true }
 env_logger = { workspace = true }
+edgehog-device-forwarder-proto = { workspace = true, optional = true }
```

Comment:
Is this needed in the `edgehog-device-runtime` package?

Reply:
No it isn't, thanks for making me notice it